src/utils/verifier.js

```diff
@@ -88,7 +94,14 @@ module.exports = {
    */
   verifyDID: async (claim) => {
     const verified = await didJWT.verifyJWT(claim)
-    return verified.payload.iss
+    const muport = verified.payload.muport
+    const res = {}
+    if (muport) {
+      const muportDID = await didJWT.verifyJWT(muport).payload.iss
```

Comment:
Same thing here. 3id-resolver will not be registered if just calling `Box.getVerifiedAccounts`.